src/App.tsx

```diff
@@ -279,48 +220,10 @@ function App(): ReactElement {
   // Add event listeners for unloading and resizing on startup.
   useEffect(() => {
     window.addEventListener("beforeunload", () => {
-      canv.domElement.removeEventListener("click", handleCanvasClick);
       canv.dispose();
     });
-
-    window.addEventListener("resize", () => {
-      setSize();
-      canv.render();
-    });
   }, []);
 
-  // CANVAS ACTIONS ///////////////////////////////////////////////////////////
-
-  const handleCanvasClick = useCallback(
-    async (event: MouseEvent): Promise<void> => {
-      const id = canv.getIdAtPixel(event.offsetX, event.offsetY);
-      // Reset track input
-      setFindTrackInput("");
-      if (id < 0) {
-        plot?.removePlot();
-        canv.setSelectedTrack(null);
-        setSelectedTrack(null); // clear selected track when clicking off of cells
-      } else {
-        const trackId = dataset!.getTrackId(id);
-        const newTrack = dataset!.buildTrack(trackId);
-        plot?.plot(newTrack, featureName, currentFrame);
-        canv.setSelectedTrack(newTrack);
-        setSelectedTrack(newTrack);
-      }
-    },
-    [dataset, featureName, currentFrame, canv, plot]
-  );
-
-  useEffect(() => {
-    canv.domElement.addEventListener("click", handleCanvasClick);
-    // Returned callback is fired if/when App is removed from the DOM,
-    // or before running useEffect again. This prevents memory issues with
-    // duplicate event listeners.
-    return () => {
-      canv.domElement.removeEventListener("click", handleCanvasClick);
-    };
-  }, [handleCanvasClick]);
-
```

Comment:
See `onTrackClicked` prop in render. Removed all references to plot here, and canvas now takes in the track as a prop, so `handleCanvasClick` reduces to two lines of code 😊